test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -686,4 +693,28 @@ public void testVarcharColumnAsMetadata() throws Exception {
     }
     testColumnTypeMetadata(content);
   }
+
+  @Test
+  public void testIntegerColumnAsMetadata() throws Exception {
+    String content = "345697";
+    executeUpdate("create table data(id int, content integer)");
+    String sql = "insert into data(id, content) values (1, ?)";
+    try (PreparedStatement ps = getConnection().prepareStatement(sql)) {
+      ps.setString(1, content);
+      assertEquals(1, ps.executeUpdate());
+    }
+    testColumnTypeMetadata(content);
+  }
+
+  @Test
+  public void testClobColumnWithNullMetadata() throws Exception {
+    String content = null;
+    executeUpdate("create table data(id int, content clob)");
+    String sql = "insert into data(id, content) values (1, ?)";
```

Comment:
Just executeUpdate("insert into data(id) values(1)") and drop the prepared statement.